frontend/app/src/components/accounts/AccountGroupHeader.vue

```diff
@@ -27,7 +27,7 @@
         <span class="font-weight-medium">
           {{ $t('account_group_header.xpub') }}
         </span>
-        <span :style="amountPrivacyStyle">
+        <span :class="!shouldShowAmount ? 'blur-content' : null">
```

Comment:
Maybe we can use the object syntax in these cases, I think when I started using this approach I was not aware of it. what do you think?

Reply:
yeah I think it would be better too.